test/js/loan_store_test.js

```diff
@@ -1060,16 +1214,80 @@ describe('Loan store tests', function() {
             var county2 = {'complete_fips': 2, 'other': 2};
             var data = [county1, county2];
             var dict = {1: county1, 2: county2};
-            
-            sandbox.spy(loanStore, 'updateLoan'); 
-            
+
+            sandbox.spy(loanStore, 'updateLoan');
+
             loanStore.updateLoanCounties(loan, data);
-            
+
             expect(loan.counties).to.equal(data);
             expect(loan['county-dict']).to.deep.equal(dict);
-            
+
         });
     });
 
+    describe('is prop linked', function() {
+      it('should correctly calculate whether loan prop is linked in current scenario', function() {
+        var getScenarioStub = sandbox.stub(scenarioStore, 'getScenario');
+        getScenarioStub.returns({});
+        var result = loanStore.isPropLinked('test');
+        expect(result).to.equal(true);
+      });
+
+      it('should correctly calculate whether loan prop is linked in current scenario, v2', function() {
+        var getScenarioStub = sandbox.stub(scenarioStore, 'getScenario');
+        getScenarioStub.returns({independentInputs: ['test']});
+        var result = loanStore.isPropLinked('test');
+        expect(result).to.equal(false);
+      });
+    });
+
+    describe('get all', function() {
+      it('should return _loans', function() {
+        var loans = [{name:'loan1', value:'loan1'}, {id:'loan2', value:'Loan #2'}];
+        loanStore._loans = loans;
+        var returned_loans = loanStore.getAll();
+        expect(returned_loans).to.deep.equal(loans);
+      });
+    });
+
+    describe('add change listener', function() {
+      it('should add a change listener', function() {
+        getAll = sandbox.stub(loanStore, 'getAll');
+        loanStore.addChangeListener(loanStore.getAll);
+        loanStore.emitChange();
+        sinon.assert.calledOnce(loanStore.getAll);
+      });
+    });
+
+    describe('remove change listener', function() {
+      it('should remove a change listener', function() {
+        getAll = sandbox.stub(loanStore, 'getAll');
+        loanStore.addChangeListener(loanStore.getAll);
+        loanStore.removeChangeListener(loanStore.getAll);
+        loanStore.emitChange();
+        sinon.assert.notCalled(loanStore.getAll);
+      });
+    });
+
+    describe('dispatch token', function() {
+      it('should do something magical', function() {
```

Comment:
I don't think it is a good idea to execute the function under test when we are not asserting anything.  It gives false data on test coverage.  So probably best to comment out the function under test so at least we know we haven't test them and will go back to them.

I think I saw that on other tests too, I am going to try to find them all.